modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -604,6 +604,29 @@
 				[/codeblock]
 			</description>
 		</method>
+		<method name="lerp_angle">
+			<return type="float">
+			</return>
+			<argument index="0" name="from" type="float">
+			</argument>
+			<argument index="1" name="to" type="float">
+			</argument>
+			<argument index="2" name="weight" type="float">
+			</argument>
+			<description>
+				Linearly interpolates between two angles (in radians) by a normalized value.
+				Similar to [func lerp] but interpolate correctly when the angles wrapped around 360 degrees.
```

Comment:
`[method lerp]`.

Reply:
Fixed

Reply:
"when the angles **wrap** around".
Also not sure if it should mention "360 degrees" if the method is in radians. Maybe mention `[constant @GDScript.TAU]`?

Reply:
Ok )